data/monster/monsters/the_last_lore_keeper.xml

```diff
@@ -0,0 +1,120 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<monster name="The Last Lore Keeper" nameDescription="The Last Lore Keeper" race="undead" experience="45000" speed="450">
+	<health now="750000" max="750000" />
+	<look type="939" corpse="0" />
```

Comment:
missing corpse since it have loot

Reply:
This boss has no corpse, loot goes directly to reward bag